scripts/apps/monitoring/directives/MonitoringGroup.ts

```diff
@@ -25,21 +33,21 @@ interface IScope extends ng.IScope {
     selected: any;
     numItems: any;
     view: any;
+    group?: StageGroup;
     viewType?: any;
-    group?: {
-        _id?: any;
-        type?: string;
-        search?: 'ingest';
-        max_items?: any;
-    };
     currentGroup: any;
     fetchNext(i: number): any;
     refreshGroup(): void;
+    customSortOptions: GroupSortOptions;
+    customSortOptionActive?: string;
```

Comment:
How about storing related data in the same field? `selectedSortOption: {field: string; direction: 'asc' | 'desc'}`